test/gui/shared/scripts/helpers/UserHelper.py

```diff
@@ -72,3 +74,23 @@ def get_displayname_for_user(username):
 
 def get_password_for_user(username):
     return get_user_info(username, "password")
+
+
+def get_username_for_user(username):
+    return get_user_info(username, "username")
+
+
+def init_predefined_users():
+    with open(os.path.abspath("../users.json"), encoding="utf-8") as f:
```

Comment:
let's allow to pass json file in like `PREDEFINED_USERS=<relative-path-to-json-file>` in the config file
```suggestion
    with open(os.path.abspath(get_config('predefined_users')), encoding="utf-8") as f:
```